source/release-notes/release-4-3-10.rst

```diff
@@ -0,0 +1,89 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: Wazuh 4.3.10 has been released. Check out our release notes to discover the changes and additions of this release.
+
+4.3.10 Release notes - 16 November 2022
+=======================================
+
+This section lists the changes in version 4.3.10. Every update of the Wazuh solution is cumulative and includes all enhancements and fixes from previous releases.
+
+What's new
+----------
+
+This release includes new features or enhancements as the following:
+
+Wazuh Kibana plugin
+^^^^^^^^^^^^^^^^^^^
+
+- Support for Kibana 7.17.6. 
+
+Wazuh Splunk app
+^^^^^^^^^^^^^^^^
+
+- Support for Splunk 8.2.7.1 and 8.2.8.
+
```

Comment:
Remove the "What's new" section. The support for Kibana 7.17.6, Splunk 8.2.7.1 and 8.2.8 was introduced in v4.3.9.